src/overlays/actors/ovl_En_Bal/z_en_bal.h

```diff
@@ -3,16 +3,40 @@
 
 #include "global.h"
 #include "z64snap.h"
+#include "objects/object_bal/object_bal.h"
+
+#define TINGLE_GET_MAP_ID(thisx) ((thisx)->params & 0xFF)
 
 struct EnBal;
 
 typedef void (*EnBalActionFunc)(struct EnBal*, PlayState*);
 
 typedef struct EnBal {
     /* 0x000 */ PictoActor picto;
-    /* 0x148 */ char unk_148[0x44];
+    /* 0x148 */ SkelAnime skelAnime;
     /* 0x18C */ EnBalActionFunc actionFunc;
-    /* 0x190 */ char unk_190[0x224];
-} EnBal; // size = 0x3B4
+    /* 0x190 */ ColliderJntSph collider;
+    /* 0x1B0 */ ColliderJntSphElement colliderElements[1];
+    /* 0x1F0 */ Vec3f balloonScale;
+    /* 0x1FC */ s32 eyeTexIndex;
+    /* 0x200 */ Vec3s jointTable[TINGLE_LIMB_MAX];
+    /* 0x2C6 */ Vec3s morphTable[TINGLE_LIMB_MAX];
+    /* 0x38C */ Vec3s headRot;
+    /* 0x392 */ Vec3s torsoRot;
+    /* 0x398 */ s16 eyeTimer;
+    /* 0x39A */ s16 forceEyesShut;
+    /* 0x39C */ s16 watchTarget;
+    /* 0x39E */ s16 skyFloatPhase;
+    /* 0x3A0 */ s16 textId;
+    /* 0x3A2 */ UNK_TYPE2 unk3A2;
```

Comment:
```suggestion
    /* 0x3A2 */ UNK_TYPE2 unk3A2;
```
```suggestion
    /* 0x3A2 */ UNK_TYPE1 unk3A2[0x2];
```
Typically for padding we always stick with `UNK_TYPE1`